src/pdf_beamtime/src/pdf_beamtime_fidpose_client.py

```diff
@@ -67,8 +67,8 @@ def main(args=None):
     rclpy.init(args=args)
 
     client = SimpleClient()
-    client.send_pickup_goal()
-    # client.send_retrun_sample_goal()
+    # client.send_pickup_goal()
+    client.send_retrun_sample_goal()
```

Comment:
`return` is misspelled here.

Reply:
Corrected:
https://github.com/maffettone/erobs/blob/db0fcb3ac75abc4ba8e9316d92be52542fbdf003/src/pdf_beamtime/src/pdf_beamtime_fidpose_client.py#L69